assets/components/contour/templates/02-service-envoy.yaml

```diff
@@ -12,9 +12,6 @@ metadata:
     # for information about enabling the PROXY protocol on the ELB to recover
     # the original remote IP address.
     service.beta.kubernetes.io/aws-load-balancer-backend-protocol: tcp
-    {{- if .Values.ingressHosts }}
-    external-dns.alpha.kubernetes.io/hostname: '{{- join "," .Values.ingressHosts }}'
```

Comment:
I have some doubts whether there should be a field in `contour` configuration that would create a DNS entry for the envoy service. The new field in contour would act as the value of the external-dns annotation.

Not sure if this is necessary, would like others to chime in.

@iaguis